packages/story-editor/src/components/library/libraryProvider.js

```diff
@@ -50,9 +50,12 @@ function LibraryProvider({ children }) {
   const [savedTemplates, setSavedTemplates] = useState(null);
   // The first page of templates to fetch is 1.
   const [nextTemplatesToFetch, setNextTemplatesToFetch] = useState(1);
+  // If to use smart colors with text and text sets.
+  const [useSmartColor, setUseSmartColor] = useState(false);
```

Comment:
I feel we should reserve the `use` prefix to custom hooks only. It might confuse new developers when this isn't a hook, but just a boolean. Maybe `shouldUseSmartColor` or `hasMagic✨` (including the emoji in the name of course).